tests/phpunit/src/Commands/Pull/PullDatabaseCommandTest.php

```diff
@@ -52,6 +45,19 @@ public function testPullDatabases(): void {
     $this->assertStringContainsString('profserv2 (default)', $output);
   }
 
+  /**
+   * Test that settings files are created for multisite DBs in IDEs.
+   *
+   * @throws \Exception
+   */
+  public function testPullDatabaseSettingsFiles(): void {
+    $this->setupPullDatabase(TRUE, TRUE, TRUE, TRUE, TRUE, TRUE);
+    $inputs = $this->getInputs();
+    putenv('AH_SITE_ENVIRONMENT=IDE');
```

Comment:
I have to set this variable manually instead of using the IDE test base, because the dedicated test base also sets an app ID and other variables that seem to conflict with the rest of this test.